### DISCUSSION TITLE
Renovate not upgrade the Golang package "gin" from 1.7.0 to 1.7.2

### DISCUSSION THREAD

**Discussion by hongbo-miao** [renovatebot/renovate#10201] [Request Help] [answered]
Golang package [gin](https://github.com/gin-gonic/gin) has new version for a while. But Renovate does not open any pull request to upgrade it from 1.7.0 to 1.7.2
https://github.com/Hongbo-Miao/hongbomiao.com/blob/main/gin/go.mod#L7

Any guide will be helpful! Thanks

**Comment by rarkins** [comment, marked as answer]
The dependency you point to is `// indirect`, which per my understanding means it's a transitive dependency. Therefore Renovate does not upgrade indirect dependencies. Previously discussed here: https://github.com/renovatebot/renovate/issues/4586

**Reply by hongbo-miao** [reply]
Thanks for the info, @rarkins !
